Dockerfile

```diff
@@ -1,34 +1,21 @@
-FROM golang:1.11-alpine
+FROM golang:1
 
 ENV PROJECT=upp-aggregate-healthcheck
 ENV ORG_PATH="github.com/Financial-Times"
 ENV SRC_FOLDER="${GOPATH}/src/${ORG_PATH}/${PROJECT}"
-
-# Set up our extra bits in the image
-RUN apk --no-cache --upgrade add \
-    git \
-    curl \
-    ca-certificates \
-    && update-ca-certificates
-
-RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+ENV BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo."
```

Comment:
This is not the correct path when using modules.

Reply:
You should use this : 
`ENV BUILDINFO_PACKAGE="github.com/Financial-Times/service-status-go/buildinfo."`